lib/resolver/javascript-file.js

```diff
@@ -27,6 +25,6 @@ export default function ({ path, target }) {
   list.push('');
 
   return list.map((file) => {
-    return 'https://github.com' + basePath + file;
+    return `https://${window.location.hostname}/${basePath + file}`;
```

Comment:
Nice workaround to avoid a local storage for the custom domain like described in #75. Anyway, I'm not sure if this still works if the permissions are less loosely again.